### ISSUE TITLE
[Feature] Make networking information visible in frontend

### ISSUE THREAD

**Issue by zcrt** [minvws/nl-kat-coordination#127]
**Is your feature request related to a problem? Please describe.**
It is not visible in the front-end of KAT from which IP scanning took and takes place.

**Describe the solution you'd like**
It would be nice to have information on the network configuration of KAT for scanning, both historically and currently. E.g. including a configurable option to show the current IP KAT would be scanning from in the front-end. Additionally, for each scan that took place it would be nice if it could be recorded what the network configuration was a time of scanning ('Did we run this boefje in country X or Y?'). 

**Describe alternatives you've considered**
Not storing this information, which is not really an alternative.

**Additional context**
This would also help preventing someone from testing KAT locally and thereby potentially impacting her/his public IP reputation

**Comment by underdarknl** [comment]
Agreed. Ideally, I think it would be useful to have a list of 'workers' on the tasklist page, with their most recent time of contact, and their settings listed. (eg, which networks can they reach). For now we are running on just one worker, but this is going to be more flexible. Each Job should be bound to the worker that executed it, and thus should be traceable to the settings for that worker.

**Comment by underdarknl** [comment]
In the future, it might be nice to use common cloud api's to generate new worker vm's when required and discard them when there is no more work left for their scope.
